lib/handleRequest.js

```diff
@@ -37,7 +37,28 @@ function handleRequest (req, res, params, context) {
     return
   }
 
-  if (method === 'OPTIONS' || method === 'DELETE') {
+  if (method === 'DELETE') {
+    if (contentType === undefined) {
+      handler(reply)
+    } else {
+      // https://github.com/fastify/fastify/issues/936
+      if (
+        contentType === 'application/json' &&
```

Comment:
Use `contentType.indexOf('application/json')` as in https://github.com/fastify/fastify/blob/e51ad47e437a7e6ecdbe65cc18cfd2f3fa57d3dc/lib/reply.js#L74 in order to be consistent

Reply:
@StarpTech Can you review it again, please?